src/web/layouts/DecideLayout.tsx

```diff
@@ -21,15 +19,6 @@ type Props = {
 	NAV: NavType;
 };
 
-// eslint-disable-next-line @typescript-eslint/no-unused-expressions
-injectGlobal`
-	/* Crude but effective mechanism. Specific components may need to improve on this behaviour. */
-	/* The not(.src...) selector is to work with Source's FocusStyleManager. */
-	*:focus {
-		${focusHalo}
-	}
-`;
-
```

Comment:
`injectGlobal` does not exist in `@emotion/react`, instead we should use `Global` which is a react component. As it needs to be rendered as a component, it makes sense to lift the rendering to the parent component (`document.tsx`)